pulp_2to3_migration/app/tasks/migrate.py

```diff
@@ -132,9 +132,9 @@ def get_repo_types(plan):
     # TODO: if plan is empty for a plugin, only migrate downloaded content
 
     pre_migrate_all_without_content(plan, type_to_repo_ids, repo_id_to_type)
+    pre_migrate_all_content(plan)
```

Comment:
@dralley , why did this move?

Reply:
I noticed that on line 205 in the other file (expand the code a couple of times) it's marking repositories as not-migrated and I figured that should happen prior to these other steps.  I think I was having issues with repository versions being kept around that shouldn't have been or something along those lines and it stopped after flipping this ordering (although I can't rule out that it was a different change I made).

I can try to flip it back and see if anything goes wrong again

Reply:
I think we fixed pulp2, so the latest pulp2 version should not require this code block on line 205, but it needs to be tested...